pkg/operator/operator.go

```diff
@@ -606,3 +588,63 @@ func (o *Operator) setMutatingWebhookCABundle(ctx context.Context, caBundle []by
 	}
 	return o.client.Update(ctx, &mwc)
 }
+
+func (o *Operator) canUpdateWebhooks(ctx context.Context) (bool, error) {
+	return o.canI(ctx,
+		authv1.ResourceAttributes{
+			Group:     arv1.GroupName,
+			Resource:  "MutatingWebhookConfiguration",
+			Namespace: o.opts.OperatorNamespace,
+			Verb:      "update",
+		},
+		authv1.ResourceAttributes{
+			Group:     arv1.GroupName,
+			Resource:  "ValidatingWebhookConfiguration",
+			Namespace: o.opts.OperatorNamespace,
+			Verb:      "update",
+		},
+	)
+}
+
+func (o *Operator) canI(ctx context.Context, resources ...authv1.ResourceAttributes) (bool, error) {
```

Comment:
nit: Can you make the name of this function more explicit? maybe merge with `canUpdateWebhooks`.

Reply:
canUpdateWebhooks is on `:592`

`canI` is a generic implementation based on the same idea as [kubectl auth can-i](https://kubernetes.io/docs/reference/access-authn-authz/authorization/#checking-api-access).